content/F2015/F15-event-career-panel.md

```diff
@@ -0,0 +1,21 @@
+title: CSC and WiCS Career Panel
+Date: 2015-09-24 17:00-19:00
```

Comment:
a 'featured' tag would definitely be helpful.
thanks for explaining - makes sense!